parsers/lib/parsers.py

```diff
@@ -21,6 +23,8 @@
     "productionPerModeForecast": PRODUCTION_PER_MODE_FORECAST_PARSERS,
     "exchange": EXCHANGE_PARSERS,
     "price": PRICE_PARSERS,
+    "dayaheadPrice": DAYAHEAD_PRICE_PARSERS,
```

Comment:
I think we might need to distinguish LMP from other price "day ahead" is partially covered by our existing "price" parser key.